css/style.css

```diff
@@ -1,3 +1,952 @@
+    border-radius: 50%;
+    width: 15px;
+    height: 15px;
+  }
+  
+  .product-rasta_b {
+    position: absolute;
+    display: inline-block;
+    right: 40px;
+    bottom: 105px;
+    background: #0e1211;
+    border-radius: 50%;
+    width: 15px;
+    height: 15px;
+  }
+  
+  .product-rasta_g {
+    position: absolute;
+    display: inline-block;
+    right: 20px;
+    bottom: 105px;
+    background: #c4c4c4;
+    border-radius: 50%;
+    width: 15px;
+    height: 15px;
+  }
+  
+  .product-rasta_y {
+    position: absolute;
+    display: inline-block;
+    right: 0px;
+    bottom: 105px;
+    background: #fdb431;
+    border-radius: 50%;
+    width: 15px;
+    height: 15px;
+  }
+
+  .products__product-camel {
+    display: none;
+  }
+
+  .products__product-mandel {
+    display: none;
+  }
+
+  .products__product-cloud {
+    display: none;
+  }
+
+  .products__img {
+    display: block;
+    width: 165.65px;
+    height: 165.42px;
+    margin: 0 auto 20px auto;
+  }
+  
+  .products__name {
+    font-size: 15px;
+    line-height: 14px;
+  }
+  
+  .products__cost {
+    font-size: 20px;
+    line-height: 19px;
+    font-weight: bold;
+  }
+
+}
+
+/* mobile 375 */
+
+@media all and (min-width:375px) {
+
+  .page {
+    max-width: 375px;
+  }
+
+  .page__header {
+    max-width: 375px;
+  }
+
+}
+
+/* tablet 768 */
+
+@media screen and (min-width:768px) {
+
+  .page {
+    max-width: 768px;
+  }
+
+  .page__header {
+    max-width: 768px;
+    height: 113.54px;
+  }
+
+  .header__logo-mobile {
+    width: 156.18px;
+    height: 33.74px;
+    left: 43px;
+    top: 41.51px;
+  }
+
+  .menu__burger {
+    width: 60.85px;
+    height: 5px;
+    top: 46.49px;
+    right: 44px;
+    border-top: 8px solid #000;
+    border-bottom: 8px solid #000;
+  }
+
+  .new-collection__goto {
+    top: 0;
+    width: 100%;
+    height: 415.28px;
+    margin: 0 auto 67.43px auto;
+  }
+
+  .new-collection__goto .goto__img-mini {
+    display: none;
+  }
+
+  .new-collection__goto .goto__img-tablet {
+    display: block;
+    width: 100%;
+    height: 415.28px;
+  }
+
+  .new-collection__goto .goto__title {
+    left: 44.42px;
+    font-size: 50px;
+    line-height: 48px;
+  }
+  
+  .goto__shop {
+    font-size: 20px;
+  }
+  
+  .goto__arrow {
+    width: 70.72px;
+  }
+
+  .products__title {
+    font-size: 30px;
+  }
+  
+  .products__list {
+    height: 352px;
+    justify-content: space-around;
+  }
+  
+  /* rasta */
+  
+  .product-rasta_r {
+    right: 75px;
+    bottom: 25px;
+    width: 20px;
+    height: 20px;
+  }
+  
+  .product-rasta_b {
+    right: 50px;
+    bottom: 25px;
+    width: 20px;
+    height: 20px;
+  }
+  
+  .product-rasta_g {
+    right: 25px;
+    bottom: 25px;
+    width: 20px;
+    height: 20px;
+  }
+  
+  .product-rasta_y {
+    right: 0px;
+    bottom: 25px;
+    width: 20px;
+    height: 20px;
+  }
+
+ /* camel */
+
+ .products__product-camel {
+   display: block;
+   align-self: flex-start;
+   text-align: left;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/eyurodin_axion/pullRequest?prid=5738041)